web_external/templates/body/submitPage.pug

```diff
@@ -28,6 +28,10 @@
   .form-group
     label Submission title
     input.c-submission-title-input.form-control(type="text", maxlength=maxTextLength)
+  .form-group
+    label Submission approach
```

Comment:
By itself, it isn't clear what an "approach" is or what should be entered in this field. I suggest adding text above that explains that a user can categorize their submissions by approach ("technique", "method" ?) and that each approach is scored independently.

Reply:
I added some text, let me know if it is sufficient to describe the intent.

Reply:
The text looks good 👍